### ISSUE TITLE
V0.6 - Watch Does not Always Reconnect after going out of Range

### ISSUE THREAD

**Issue by jones139** [OpenSeizureDetector/PineTimeSD#15]
Sometimes the watch does not reconnect when it goes out of range then back again - phone will stay in FAULT state until bluetooth is re-started on the watch.

**Comment by jones139** [comment]
When this happened for me, I had noticed that even when everything was working, the bluetooth icon was not displayed in the top right hand corner of the watch screen.   The icon reappeared after switching bluetooth off and on again to re-connect the watch to the phone.

**Comment by seaside1** [comment]
I have experienced the same.

**Comment by jones139** [comment]
A key thing to check next time this happens is whether re-starting the phone app cures it or not, or if you do have to do something on the watch - it is much easier for me to re-start the phone app automatically if that is useful.

**Comment by seaside1** [comment]
I'll check I've been using two different android phones wiith different android versions (major), and as I recall I could restart the app on one of them but not the other.

**Comment by jones139** [comment]
I am having a bit of trouble persuading the BLE data source to re-start without leaving some parts of it running and messing things up - should hopefully get it in a couple of days....

**Comment by seaside1** [comment]
Given what is written in https://github.com/InfiniTimeOrg/InfiniTime/issues/895 what is the most recent version of the PinTime FW to use? I would like to test out PineTime again.

**Comment by jones139** [comment]
Sorry for the delay.   We have been running V0.7 for a few months on our 'production' seizure detector set up and it is working ok - I have just updated github to show V0.7 as the latest release, so I'd suggest trying that.

I want to write a little article on our experiences of using pinetime when I get chance - the main issue we have had is that the connection reliability is very phone dependent.    We started off using a new Android Go phone (Moto E13 I think) and this worked fine for a couple of months, then suddenly started dropping connections unless the phone was very close to the watch (like right next to the bed).   I don't know if it developed a hardware fault or if it was a software update that messed it up.   So we are now using my old Samsung A20e and it is working fine with V0.7 (note that it appears as Version 230.0.7 for reasons to do with making it work with Gadgetbridge without having to do a custom build of Gadgetbridge).
